linux_os/guide/system/permissions/restrictions/coredumps/coredump_disable_storage/tests/coredumps_storage_none_dropin.pass.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
This test scenario doesn't pass on RHEL8, it fails at the setup phase, I think you can fix it by creating the directory beforehand.

```suggestion
mkdir -p /etc/systemd/coredump.conf.d
```